website/docs/migration_guide.html.markdown

```diff
@@ -0,0 +1,135 @@
+---
+layout: "scaleway"
+page_title: "Migration Guide"
+description: |-
+  Migrating your Scaleway provider from v1 to v2.
+---
+
+# Migrating from v1 to v2
+
+This page guides you through the process of migrating your version 1 resources to their version 2 equivalent.
+In version 2 we completely changed the naming of all resources, as well as their attributes.
```

Comment:
Introduce some context like:
  - To prepare the launch of all new scaleway product we completely changed the naming....